gdnative-core/src/core_types/vector3.rs

```diff
@@ -1,5 +1,36 @@
 use crate::Vector3;
 
+#[derive(Copy, Clone, Debug, PartialEq, Eq, PartialOrd, Ord)]
+pub struct Axis(pub i64);
```

Comment:
I still think that `Axis` should continue to be a real Rust enum, because there isn't room for error here: it's not possible for `Axis` to be bitflags or have duplicated variants, because it's implemented by hand!

Reply:
Axis is back, just not in the top level.